core/arrayUtils.js

```diff
@@ -16,7 +16,10 @@ const removeItemAtIndex =
   (array) =>
     [...array.slice(0, index), ...array.slice(index + 1)]
 
+const fromNumberOfElements = (numOfElements) => Array.from(Array(numOfElements).keys())
```

Comment:
create? I am not sure about the name of the function

Reply:
I've been inspired by the Array function "from" function...